madgiraffestudios/src/components/Booking/Booking.js

```diff
@@ -53,7 +53,8 @@ const [message, setMessage] = useState ('');
         title: "Message on the way!",
         text: `We can't wait to work with you, ${fullName}!`,
         icon: "success",
-        dangerMode: false
+        timer: 3000
```

Comment:
- [ ] This should be added to the failure message as well and perhaps set the timeout for 5000 for the error to give the client enough time to read the information and grab the email

Reply:
- [ ] can you pass this as a parameter after timer -- to get rid of the OK buttons
`buttons: {
          cancel: null
        }`